docsets/XMake/docset.json

```diff
@@ -9,8 +9,8 @@
     "aliases": ["Xmake"],
     "specific_versions": [
         { 
-            "version": "2.2.8",
-            "archive": "versions/2.2.8/XMake.tgz"
+            "version": "2.2.9",
+            "archive": "docsets/XMake/XMake.tgz"
```

Comment:
This doesn't work. The specific version path can't be the same as the main docset path.

The main docset path (docsets/XMake/Xmake.tgz) is still unchanged.

Reply:
And now it works?
>docsets/XMake/versions/2.2.9/Xmake.tgz